src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,6 +49,113 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants">
```

Comment:
A couple of comments here:

1) Lots of these properties are C#-specific, we should break these out into a C# version, VB version and leave shared set between C# and VB.
2) I think we should pull out compiler options into a separate rule to separate it from "general" properties.